flycheck.el

```diff
@@ -4170,7 +4193,14 @@ non-nil."
     (setq flycheck-display-error-at-point-timer
           (run-at-time flycheck-display-errors-delay nil 'flycheck-display-error-at-point))))
 
-
+(defun flycheck-try-fix-error-at-point ()
+  "Attempt to fix error at point using the list of fixer functions."
+  (interactive)
+  (let ((err (car-safe (flycheck-overlay-errors-at (point)))))
+    (if err
+        (run-hook-with-args-until-success 'flycheck-fix-error-functions err)
+      (message "No errors at point to fix!"))))
```

Comment:
my thought was that it may not necessarily be the case that multiple errors can be fixed in one pass, because the mutation as a result of fixing the first error might invalidate the latter errors (at least the line/column information, for instance).